content/en/blog/2019/istioctl-describe/index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "'istioctl describe' in Istio 1.3"
+description: Introducing a command-line utility to find configuration affecting pod traffic.
+publishdate: 2019-09-20
+attribution: Ed Snible (IBM)
+keywords: [traffic-management, istioctl, debugging]
+---
+
+Istio 1.3 includes an experimental command-line tool designed to help find and
+understand configuration that affects traffic to a pod.
+
+`istioctl experimental describe pod <pod>` shows the Istio configuration that affects a pod.
+
+The team has prepared a mini-tutorial to demonstrate the features and use of the tool.
+
+## describe tutorial using Bookinfo tutorial
```

Comment:
This content should not be part of the blog post. Please create a task for this feature. See our [guide on page types](https://istio.io/about/contribute/creating-and-editing-pages/#choosing-a-page-type) for more details. 
@esnible I'll be happy to help you or any other member of the WG polish this content for publication.